src/plugins/intel_cpu/src/emitters/x64/jit_eltwise_emitters.cpp

```diff
@@ -2243,5 +2243,189 @@ void jit_select_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const
         h->vblendmps(vmm_dst | k_mask, vmm_src1, vmm_src0);
     }
 }
+
+/// BITWISE_AND ///
+jit_bitwise_and_emitter::jit_bitwise_and_emitter(x64::jit_generator* host, x64::cpu_isa_t host_isa, const std::shared_ptr<ov::Node>& node, Precision exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {}
+
+jit_bitwise_and_emitter::jit_bitwise_and_emitter(x64::jit_generator* host, x64::cpu_isa_t host_isa, Precision exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {}
+
+size_t jit_bitwise_and_emitter::get_inputs_num() const { return 2; }
+
+std::set<std::vector<element::Type>> jit_bitwise_and_emitter::get_supported_precisions(const std::shared_ptr<ngraph::Node>& node) {
+    return {
+        {element::i8, element::i8},
+        {element::i16, element::i16},
+        {element::i32, element::i32},
+        {element::u8, element::u8},
+        {element::u16, element::u16}
+    };
+}
+
+void jit_bitwise_and_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == x64::sse41) {
+        emit_isa<x64::sse41>(in_vec_idxs, out_vec_idxs);
+    } else if (host_isa_ == x64::avx2) {
+        emit_isa<x64::avx2>(in_vec_idxs, out_vec_idxs);
+    } else if (host_isa_ == x64::avx512_core) {
+        emit_isa<x64::avx512_core>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_ASSERT(!"unsupported isa");
+    }
+}
+
+template <x64::cpu_isa_t isa>
+void jit_bitwise_and_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const {
+    using Vmm = typename conditional3<isa == x64::sse41, Xmm, isa == x64::avx2, Ymm, Zmm>::type;
+    Vmm vmm_src0 = Vmm(in_vec_idxs[0]);
+    Vmm vmm_src1 = Vmm(in_vec_idxs[1]);
+    Vmm vmm_dst = Vmm(out_vec_idxs[0]);
+
+    h->uni_vandps(vmm_dst, vmm_src0, vmm_src1);
```

Comment:
This will not work on SSE in case vmm_dst != vmm_src0

Reply:
fixed, thanks!